src/components/pages/home/index.tsx

```diff
@@ -114,34 +117,27 @@ const Home: FunctionComponent<any> = ({homePageData}) => {
                     />
                   </div>
                   <div className="max-w-screen-sm space-y-3">
-                    <h1 className="md:text-3xl text-2xl dark:text-gray-200 font-bold leading-tight">
-                      Digital Gardening for Developers
-                    </h1>
+                    <Link href={featureDigitalGardening.path}>
```

Comment:
`digitalGardeningQuery` passes the path to the subpage `/learn/digital-gardening`